yml/cdb_startrackerfast.yaml

```diff
@@ -0,0 +1,894 @@
+  - name: group_id
+    "@id": "#visit1.group_id"
+    datatype: string
+    length: 64
+    description: Identifier for the group that this visit is part of.
+  - name: cur_index
+    "@id": "#visit1.cur_index"
+    datatype: int
+    description: Number (1-based) of the observation within its group.
+  - name: max_index
+    "@id": "#visit1.max_index"
+    datatype: int
+    description: Expected number of observations within the group.
+  - name: img_type
+    "@id": "#visit1.img_type"
+    datatype: string
+    length: 64
+    description: Type of visit taken.
+  - name: science_program
+    "@id": "#visit1.science_program"
+    datatype: string
+    length: 64
+    description: Science program.
+  - name: observation_reason
+    "@id": "#visit1.observation_reason"
+    datatype: string
+    length: 68
+    description: Reason for the observation.
+  - name: target_name
+    "@id": "#visit1.target_name"
+    datatype: string
+    length: 64
+    description: Target of the observation.
+  - name: air_temp
+    "@id": "#visit1.air_temp"
+    datatype: float
+    description: Outside air temperature in degC.
+    ivoa:ucd: phys.temperature
+  - name: pressure
+    "@id": "#visit1.pressure"
+    datatype: float
+    description: Outside air pressure.
+    ivoa:unit: Pa
+  - name: humidity
+    "@id": "#visit1.humidity"
+    datatype: float
+    description: Outside relative humidity.
+  - name: wind_speed
+    "@id": "#visit1.wind_speed"
+    datatype: float
+    description: Outside wind speed.
+    ivoa:unit: m/s
+  - name: wind_dir
+    "@id": "#visit1.wind_dir"
+    datatype: float
+    description: Wind direction.
+    ivoa:unit: deg
+  - name: dimm_seeing
+    "@id": "#visit1.dimm_seeing"
+    datatype: float
+    description: Seeing as measured by external DIMM (FWHM).
+    ivoa:unit: arcsec
+  - name: shut_lower
+    "@id": "#visit1.shut_lower"
+    datatype: float
+    description: Dome dropout door opening percentage.
+  - name: shut_upper
+    "@id": "#evisit1.shut_upper"
+    datatype: float
+    description: Dome main door opening percentage.
+  - name: focus_z
+    "@id": "#visit1.focus_z"
+    datatype: float
+    description: Focus Z position.
+  - name: simulated
+    "@id": "#visit1.simulated"
+    datatype: boolean
+    description: Were any control system components simulated?
+  - name: vignette
+    "@id": "#visit1.vignette"
+    datatype: string
+    length: 10
+    description: "Instrument blocked from the sky: UNKNOWN, NO, PARTIALLY, FULLY."
+  - name: vignette_min
+    "@id": "#visit1.vignette_min"
+    datatype: string
+    length: 10
+    description: "Lowest amount of instrument vignetting detected during the visit: UNKNOWN, NO, PARTIALLY, FULLY."
+  - name: s_region
+    "@id": "#visit1.s_region"
+    datatype: string
+    length: 1024
+    description: Sky region in STC-S format (https://www.ivoa.net/documents/STC-S/20130917/index.html).
+    ivoa:ucd: pos.outline;obs.field
+- name: visit1_quicklook
```

Comment:
Rename this one to `exposure_quicklook` and link it to the exposure table instead of the visit one.